googletest/src/gtest-internal-inl.h

```diff
@@ -390,10 +390,6 @@ class GTEST_API_ UnitTestOptions {
   // This function is useful as an __except condition.
   static int GTestShouldProcessSEH(DWORD exception_code);
 #endif  // GTEST_OS_WINDOWS
-
-  // Returns true if "name" matches the ':' separated list of glob-style
-  // filters in "filter".
-  static bool MatchesFilter(const std::string& name, const char* filter);
```

Comment:
Unfortunately, it turns out we have some internal code (i.e. not published to GitHub) that uses this static method.
Instead of deleting it, could we reimplement it using the new APIs?